	Add virtual destructors to avoid compiler warning.

	May actually be required, or just be a warning?

diff -ur dvisvgm-0.8.7.orig/src/CmdLineParserBase.h dvisvgm-0.8.7/src/CmdLineParserBase.h
--- dvisvgm-0.8.7.orig/src/CmdLineParserBase.h	2009-11-06 22:17:58.000000000 +0100
+++ dvisvgm-0.8.7/src/CmdLineParserBase.h	2009-11-13 18:55:12.000000000 +0100
@@ -72,6 +72,7 @@
 	protected:
 		CmdLineParserBase () : _error(false) {}
 		CmdLineParserBase (const CmdLineParserBase &cmd) {}
+		virtual ~CmdLineParserBase () {}
 		virtual void init ();
 		virtual void error (const Option &opt, bool longopt, const char *msg) const;
 		bool checkArgPrefix (InputReader &ir, const Option &opt, bool longopt) const;
diff -ur dvisvgm-0.8.7.orig/src/CommandLine.h dvisvgm-0.8.7/src/CommandLine.h
--- dvisvgm-0.8.7.orig/src/CommandLine.h	2009-11-06 22:22:55.000000000 +0100
+++ dvisvgm-0.8.7/src/CommandLine.h	2009-11-13 18:55:09.000000000 +0100
@@ -14,6 +14,7 @@
    public:
       CommandLine () {init();}
       CommandLine (int argc, char **argv, bool printErrors) {parse(argc, argv, printErrors);}
+      virtual ~CommandLine () {}
       void help () const;
       void status () const;
       int numOptions () const {return 22;}
